rxlifecycle-android-lifecycle/src/main/java/com/trellow/relifecycle2/android/lifecycle/RxLifecycleAndroidLifecycle.java

```diff
@@ -0,0 +1,45 @@
+package com.trellow.relifecycle2;
+
+import android.arch.lifecycle.Lifecycle;
+import android.support.annotation.NonNull;
+
+import com.trello.rxlifecycle2.LifecycleTransformer;
+import com.trello.rxlifecycle2.OutsideLifecycleException;
+
+import io.reactivex.Observable;
+import io.reactivex.functions.Function;
+
+import static com.trello.rxlifecycle2.RxLifecycle.bind;
+
+public class RxLifecycleAndroidLifecycle {
+
+    private RxLifecycleAndroidLifecycle() {
+        throw new AssertionError("No instances");
+    }
+
+    public static <T> LifecycleTransformer<T> bindLifecycle(@NonNull Observable<Lifecycle.Event> lifecycle) {
+        return bind(lifecycle, LIFECYCLE);
+    }
+
+    private static final Function<Lifecycle.Event, Lifecycle.Event> LIFECYCLE = new Function<Lifecycle.Event, Lifecycle.Event>() {
+        @Override
+        public Lifecycle.Event apply(Lifecycle.Event lastEvent) throws Exception {
```

Comment:
I chose to go with the provided events instead of the `rxlifecycle-android` ones. They are a bit more limited on fragments so I think this is less surprizing.